packages/common/src/resourceToLabel.ts

```diff
@@ -1,9 +1,9 @@
-const rootNamespace = "";
+export const ROOT_NAMESPACE = "";
```

Comment:
we should probably lift this out into its own file and use it more throughout the TS codebase for clarity

Reply:
Absolutelyy, I use this a lot